crates/cheatcodes/spec/src/vm.rs

```diff
@@ -2946,6 +2946,13 @@ interface Vm {
     /// Generates a ready-to-sign digest of human-readable typed data following the EIP-712 standard.
     #[cheatcode(group = Utilities)]
     function eip712HashTypedData(string calldata jsonData) external pure returns (bytes32 digest);
+
+    /// RLP encodes a list of bytes into an RLP payload.
+    #[cheatcode(group = Utilities)]
+    function toRlp(bytes[] calldata data) external pure returns (bytes memory);
+    /// RLP decodes an RLP payload into a list of bytes.
+    #[cheatcode(group = Utilities)]
+    function fromRlp(bytes calldata rlp) external pure returns (bytes[] memory data);
```

Comment:
unsure how useful this really is because this can only bytes

but imo these dont really hurt

any opinions here @grandizzy